app/views/dashboard/dashboard.html.erb

```diff
@@ -105,3 +105,19 @@
         </div>
     </div>
 </div>
+
+<h5 class="mb-3 text-dark font-weight-bold">RECOMMENDED VIDEOS</h5>
+<div class="overflow-auto">
+  <ul class="list-unstyled video-list-thumbs d-flex">
+    <% Youtube.search(current_teacher.subject).each do |video| %>
```

Comment:
Hi @pielagomrt  Please use the youtube module inside a controller, store it inside an instance variable, and then loop using that instance variable from the controller